### ISSUE TITLE
Downloading attachments with non-ascii characters is broken with Python 2

### ISSUE THREAD

**Issue by cXhristian** [django-wiki/django-wiki#383]
```
UnicodeDecodeError at /12/plugin/attachments/download/33/
'ascii' codec can't decode byte 0xc3 in position 0: ordinal not in range(128)
Request Method:	GET
Request URL:	http://localhost:8000/12/plugin/attachments/download/33/
Django Version:	1.7.4
Exception Type:	UnicodeDecodeError
Exception Value:	
'ascii' codec can't decode byte 0xc3 in position 0: ordinal not in range(128)
Exception Location:	/home/christian/Projects/Python/django-wiki/testproject/wiki/core/http.py in send_file, line 50
Python Executable:	/home/christian/.virtualenvs/wiki-py2/bin/python
```

**Comment by cXhristian** [comment]
The cause is `from __future__ import unicode_literals`. 

I think the fix is to remove `.encode('utf-8')`, but I don't know enough about unicode handling in Python2/3 to be sure about this.

**Comment by benjaoming** [comment]
Thanks, this is now released in 0.0.24.2
